README.rst

```diff
@@ -161,6 +161,43 @@ fault by adding the ``-v`` (verbose) flag::
 
 I need to rebuild this wheel to link with dual-architecture libraries.
 
+Making dual-architecture binaries
+=================================
+
+Modern Mac wheels can be either ``arm64e`` (M1 ARM), ``x86_64`` (64-bit Intel) or
+both ("universal").
+
+Building an entire Python wheel as dual-architecture can be difficult, perhaps
+because you need to link different libraries in the two cases, or you need
+different compiler flags.
+
+One solution to this problem is to do an entire ``arm64e`` wheel build, and then
+an entire ``x86_64`` wheel build, and *fuse* the two wheels into a universal
+wheel.
+
+That is what the ``delocate-fuse`` command does.
+
+Let's say you have built an ARM and Intel wheel, called, respectively:
+
+* ``scipy-1.9.3-cp311-cp311-macosx_12_0_arm64.whl``
+* ``scipy-1.9.3-cp311-cp311-macosx_10_9_x86_64.whl``
```

Comment:
consider replacing `scipy` with `pkgname` for a bit more genericity, and `12_0` with `11_0` because that'd be the normal tag (SciPy is special here, because its test suite triggers a kernel panic on 11.0).

Reply:
I've used Scipy as an example a lot above, so I'll stick with Scipy here - otherwise I think it would be more confusing.  I've changed to `11_0`.